### PR TITLE
Metadata handling

### PR DESCRIPTION
**Pull request by Joshocan**

### CODE REVIEW

File: metadata/oai-pmh-provider-guidelines.md [hunk 1]
```diff
@@ -6,57 +6,85 @@ nav_order: 720
...
+[OVAL BASE](https://oval.base-search.net).
 
 ## Metadata Requirements
 
-Data Providers must provide an endpoint exposing metadata represented as DDI 2.5 Codebook.
-The DDI files must also be compliant with the [CESSDA Metadata Model (CMM)](https://zenodo.org/record/3543756).
-The compliance with CMM can be checked using the [CESSDA Metadata Validator](https://cmv.cessda.eu).
-We recommend validating some of your metadata to expose potential systemic issues before making your endpoint available.
+Service (Data) Providers must provide an endpoint exposing metadata represented
+as DDI 2.5 Codebook standard.Looking ahead, the CDC is set to expand its
```
Comment (john-shepherdson) [inline metadata/oai-pmh-provider-guidelines.md:54]: standard.Looking -> standard. Looking

File: metadata/oai-pmh-provider-guidelines.md [hunk 2]
```diff
@@ -6,57 +6,85 @@ nav_order: 720
...
+support to include the DDI 3.2 Lifecycle format,furthering enhancing its
```
Comment (john-shepherdson) [inline metadata/oai-pmh-provider-guidelines.md:55]: DDI 3.2 Lifecycle -> DDI Lifecycle

**Comment by Joshocan** [comment]
Linting failure: The sentence on the list is too long. Trying to resolve violation MD032, results in violation MD013. This can be made to pass.

**Comment by john-shepherdson** [comment]
From Jenkins:

./metadata/oai-pmh-provider-guidelines.md:66: MD032 Lists should be surrounded by blank lines
./metadata/oai-pmh-provider-guidelines.md:68: MD032 Lists should be surrounded by blank lines
./metadata/urn.md:16: MD032 Lists should be surrounded by blank lines
./metadata/urn.md:18: MD032 Lists should be surrounded by blank lines

**Comment by Joshocan** [comment]
@john-shepherdson Linting failure: The sentence on the list is too long for the list. Trying to resolve violation **MD032** results in violation **MD013**. Unless we shorten (summarise) the sentences for each, which might change the context.

**Comment by john-shepherdson** [comment]
Which branch are you working on?

**Comment by Joshocan** [comment]
[Metadata-handling](https://github.com/cessda/cessda.guidelines.public/tree/Metadata-handling)

**Comment by john-shepherdson** [comment]
What I see in urn.md lines 16 to 18 is

* `int.cessda.cv` - CESSDA owned vocabularies, *not* the service
* `int.cessda.elsst` - ELSST Thesaurus
* `int.cessda.eqb` - EQB

and oai-pmh-provider-guidelines.md lines 62 to 65:

- URL of the OAI-PMH endpoint (e.g. `https://xxx.xxx.xxx/oai-pmh`)
- Metadata prefix (e.g. `oai_ddi25`)
- Set specification, if relevant (e.g. `subject:social-sciences`)
- Name and email address of the person responsible for the endpoint.

So nothing too long there

**Review by john-shepherdson — APPROVED** [review]

**Pull request merged**